### ISSUE TITLE
Incompatible with torrent tags

### ISSUE THREAD

**Issue by the-white-light** [fedarovich/qbittorrent-cli#6]
Version 1.2.18320.1

When attempting to run `qbt torrent list` I get the following error:

```
Error converting value "Force" to type 'System.Collections.Generic.IReadOnlyCollection`1[System.String]'. Path '[127].tags', line 1, position 188543.
Could not cast or convert from System.String to System.Collections.Generic.IReadOnlyCollection`1[System.String].
```

The only thing I can think of that would have "Force" is some torrents that have been tagged.

**Comment by fedarovich** [comment]
And what version of qBittorrent do you use?

**Comment by the-white-light** [comment]
I'm using qb4.1.4

**Comment by fedarovich** [comment]
Fixed in version 1.2.18360.1
